operators/vision/draw_bounding_box.hpp

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "ocos.h"
+#include "onnxruntime_c_api.h"
+#include "string_utils.h"
+
+#include <cstdint>
+
+namespace ort_extensions {
```

Comment:
may we unify the root namespace name as OrtX, more consistent with Ort style?
@skottmckay

Reply:
We should definitely unify.

ORT follows the Google C++ Style Guide so namespace should be lowercase with '_' separator, and the name should be clear and not unnecessarily abbreviated.

https://google.github.io/styleguide/cppguide.html#Namespace_Names
https://google.github.io/styleguide/cppguide.html#General_Naming_Rules

`ort_extensions` seems to fit those rules. I think the most you could shorten that is to `ort_ext` but that's possibly getting into unnecessarily abbreviated territory.

Reply:
any story about this Ort in this C++ api?
https://github.com/microsoft/onnxruntime/blob/2ee822d4831d2043e3d588d601264ed96cd72bf3/include/onnxruntime/core/session/onnxruntime_cxx_api.h#L45, 
And I remember I also saw this pattern in ort codebase for multiple times and take it as de facto standard

ort-ext is not good since it neither be short enough for convenience, nor precise of full name.

Reply:
IIRC we had to use `Ort` as the namespace for the public C++ API couldn't clash with the existing `onnxruntime` namespace used in all the other C++ code. 

I'm not aware of any complaints about the convenience of the `onnxruntime` namespace and `ort_extensions` is a pretty similar length. I don't find myself typing it very often - mainly a couple of times when adding a new class, once in the header and once in the .cc, and auto-complete usually means only a few characters actually need to be typed. 

If we want to be consistent with the style guide and internal ORT code that would be `ort_extensions` wouldn't it?